orchestration/dagster/hca_orchestration/solids.py

```diff
@@ -0,0 +1,66 @@
+from dagster import solid, Nothing, InputDefinition, ExpectationResult, String
```

Comment:
This file houses the solids used in our pipelines, see https://docs.dagster.io/overview/solids-pipelines/solids for more details.